ecies.c

```diff
@@ -19,14 +19,6 @@ const char key[] = \
 "nzwv/wzEUsNPrLSUfMq1dYvt3hk=\n" \
 "-----END PRIVATE KEY-----\n";
 
-void RAND_init(void) {
```

Comment:
Also, from OpenSSL docs, "OpenSSL makes sure that the PRNG state is unique for each thread. On systems that provide /dev/urandom, the randomness device is used to seed the PRNG transparently. However, on all other systems, the application is responsible for seeding the PRNG by calling RAND_add(), RAND_egd or RAND_load_file." https://www.openssl.org/docs/man1.0.2/crypto/RAND_add.html